js/page-specific/swiftype-search-results.js

```diff
@@ -6,19 +6,40 @@ baltimoreCounty.pageSpecific.swiftypeSearchResults = (function swiftypeSearchRes
 	var templateSelector = '#swiftype-search-results-template';
 	var errorMessageHtml = '<p>There were no results found for this search.</p>';
 
-	function getSearchResults(searchTerm, pageNumber) {
-		var currentPageNumber = pageNumber || 1;
-		var cleanedSearchTerm = searchTerm.trim();
-
+	function cleanSearchTerm(termToClean) {
+		var cleanedSearchTerm = termToClean.trim();
+		var safeSearchTerms = [];
 		// Ensure the last character is not '+' as the the trailing space causes no results
 		if (cleanedSearchTerm[cleanedSearchTerm.length - 1] === '+') {
 			cleanedSearchTerm = cleanedSearchTerm.slice(0, -1);
 		}
 
-		cleanedSearchTerm = cleanedSearchTerm.replace(/\+/g, '%20');
+		var searchTerms = cleanedSearchTerm.split('+');
+		for (var i = 0; i < searchTerms.length; i += 1) {
+			var term = decodeURIComponent(searchTerms[i]);
+			term = term
+				.replace(/\?/g, '')
+				.replace(/#/g, ' ')
+				.replace(/&/g, 'and')
+				.replace(/\//g, ' ')
+				.replace(/\\/g, ' ');
+
+			var encodedTerm = encodeURIComponent(term);
+			safeSearchTerms.push(encodedTerm);
+		}
+		cleanedSearchTerm = safeSearchTerms.join('+');
+
+		return cleanedSearchTerm.replace(/\+/g, '%20');
```

Comment:
Should this be done inside your loop now?

Reply:
The for loops excludes the + symbol, but I can change this.